### DISCUSSION TITLE
Support Extra Channels in TIFF and PSD

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#6709] [Contribute]
Sorry, I am not familiar with posting here. Dirk made a request to post this in relation to 
https://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=34473

The text file is linked there, but here is a copy of the link 
http://www.tecnogravura.com/fmw42.7z

If you need help with Photoshop testing, I am available.

Fred

**Comment by vadimaus** [comment]
Hello! 
Is there any way to compose individual layers and make a PSD file? I'm using the latest PHP Imagick library. Thanks

**Comment by fmw42** [comment]
Your question has nothing to do with extra channels. You are asking about layers. Nevertheless, see https://www.php.net/manual/en/imagick.addimage.php

**Comment by dlemstra** [comment]
This has been implemented so I will close this discussion.
